app/controllers/idv/in_person/usps_locations_controller.rb

```diff
@@ -26,7 +26,9 @@ def index
           city: search_params['city'], state: search_params['state'],
           zip_code: search_params['zip_code']
         )
-        response = proofer.request_facilities(candidate)
+        is_enhanced_ipp = resolved_authn_context_result.enhanced_ipp? &&
+                          Identity::Hostdata.env != 'prod'
```

Comment:
@gina-yamada I know you wanted us to be sure that prod is not affected.  Does this environment check address your concerns?  Let me know, thanks!

Reply:
Is this here to prevent the feature from being accidentally enabled in prod? What is the risk of a feature flag being flipped from a default accidentally versus the risk of this lingering and the feature not being enabled in production when we toggle the config to say that it is?

Reply:
I am curious to hear from @gina-yamada in response to this question.  I'm still formulating my thoughts.

Reply:
Yes, not necessary -- suggestion to be out of abundance of caution.  Feel free to remove this additional check

Reply:
I will remove the check in my next commit.